src/Brick/ReflexMain.hs

```diff
@@ -157,37 +159,51 @@ brickWrapper shouldHaltE widgetDyn cursorDyn attrDyn = do
             $ ((io >>= resultH) `finally` restartH ())
         return resultEvent
 
+  rsRef <- liftIO $ newIORef initialRS
+
+  let startOrSuspend :: These () () -> Maybe (Vty, IO ()) -> R.PushM t (Maybe (Vty, IO ()))
+      startOrSuspend = \case
+        This{} -> \_ -> do
+          widgetStack  <- R.sample $ R.current widgetDyn
+          chooseCursor <- R.sample $ R.current cursorDyn
+          attrs        <- R.sample $ R.current attrDyn
+          vty <- liftIO $ do
+            x <- mkVty defaultConfig
+            return x
+          liftIO $ do
+            renderState <- readIORef rsRef
+            (renderState', _) <-
+              render vty widgetStack chooseCursor attrs renderState
+            writeIORef rsRef renderState'
+            
+          let loop = forever $ do
+                ev <- atomically
+                  (readTChan $ _eventChannel $ inputIface vty)
+                case ev of
+                  (EvResize _ _) ->
+                    eventH
+                      .   Just
+                      .   (\(w, h) -> EvResize w h)
+                      =<< (displayBounds $ outputIface vty)
+                  _ -> eventH $ Just ev
+          pumpTId <- liftIO $ forkIO $ loop
+          liftIO . void . forkIO . void $ eventH Nothing
```

Comment:
There is no need to have this event be a `Maybe` anymore, as the purpose of the `Nothing` there was to signal a redrawing request - or do we still want this?

i.e. i think we can change `R.Event t (Maybe Event)` to `R.Event t Event` in [this line](https://github.com/LightAndLight/bricki-reflex/blob/2110d3709447d396418c76bd9de58db52cbb9bf0/src/Brick/ReflexMain.hs#L129) and remove the nothing firing.